encrypted-media-respec.html

```diff
@@ -2089,15 +2080,12 @@
           </ol>
         </section>
 
-        <section id="session-close">
-          <h4>Session Close</h4>
-          <p>The Session Close algorithm is run when the CDM closes the session associated with a <a>MediaKeySession</a> object.</p>
+        <section id="close-session">
```

Comment:
Ok, I will roll in a couple of the other issues into this PR. Specifically, make things consistent in terms of "use the CDM to..." and have a small and consistent set of things that we use the CDM to do with respect to the session:
- close the session (which destroys all non-persistent licenses and keys)
- destroy persisted licenses and keys
- destroy all persisted state (record of license destruction, record of key usage)